javatests/arcs/core/storage/CapabilitiesResolverTest.kt

```diff
@@ -38,71 +45,71 @@ class CapabilitiesResolverTest {
         CapabilitiesResolver.defaultCreators.clear()
     }
 
+    fun verifyStorageKey(key: StorageKey?, expectedClass: Class<out StorageKey>) {
+        assertThat(key).isInstanceOf(ReferenceModeStorageKey::class.java)
+        assertThat((key as ReferenceModeStorageKey).backingKey).isInstanceOf(expectedClass)
+        assertThat(key.storageKey).isInstanceOf(expectedClass)
+    }
+
     @Test
     fun capabilitiesResolver_createsStorageKeysDefault() {
-        val options = CapabilitiesResolver.StorageKeyOptions(ArcId.newForTest("test"))
+        val options = CapabilitiesResolver.CapabilitiesResolverOptions(ArcId.newForTest("test"))
         val resolver = CapabilitiesResolver(options)
         assertThat(resolver.findStorageKeyProtocols(Capabilities.TiedToArc))
             .containsExactly(VOLATILE_DRIVER_PROTOCOL)
         assertThat(resolver.findStorageKeyProtocols(Capabilities.TiedToRuntime)).isEmpty()
         assertThat(resolver.findStorageKeyProtocols(Capabilities.Persistent)).isEmpty()
-        assertThat(resolver.createStorageKey(Capabilities.TiedToArc))
-            .isInstanceOf(VolatileStorageKey::class.java)
+        verifyStorageKey(resolver.createStorageKey(Capabilities.TiedToArc, thingSchema, handleId), VolatileStorageKey::class.java)
```

Comment:
I think this line goes over 100 chars of width?

Reply:
fixed